raw/mock_exam/docs/questions/question-37.adoc

```diff
@@ -0,0 +1,92 @@
+// tag::DE[]
+=== Frage 37
+**ID: Q-20-04-28**
+
+[cols="2,8,2", frame=ends, grid=rows]
+|===
+| P-Frage: 
+| Wählen Sie die **vier** am besten passenden Antworten aus
+| 1 Punkt
+|===
+
+Welche vier der folgenden Punkte sind am besten geeignet, eine qualitative Analyse Ihrer Softwarearchitektur zu unterstützen?
+
+[cols="1a,1,10", frame=none, grid=none]
+|===
+
+| {y}
+| (a)
+| Quantitative Abhängigkeitsanalyse.
+
+| {y}
+| (b)
+| Architekturmodelle.
+
+| {y}
+| (c)
+| Qualitätsszenarien.
+
+| {n}
+| (d)
+| Teamgröße.
```

Comment:
Würde ich entfernen, da es mit Team Topologies in diese Richtung gehen kann

Reply:
P-question, therefore it's imho ok